configuration.rst

```diff
@@ -344,7 +344,7 @@ files are always in sync.
 
     When you :ref:`install an existing Symfony project <install-existing-app>`, you
     will need to create the ``parameters.yml`` file using the committed ``parameters.yml.dist``
-    file as a reference. To help with this, after you run ``composer install``, a
+    file as a reference. To help with this, after you run ``$ composer install``, a
```

Comment:
For commands displayed inside a paragraph we don't include the leading `$`, only for proper console blocks.